bin/mpdevil

```diff
@@ -3762,7 +3762,7 @@ class MainWindow(Gtk.ApplicationWindow):
 		self.set_default_size(settings.get_int("width"), settings.get_int("height"))
 		if settings.get_boolean("maximize"):
 			self.maximize()  # request maximize
-		Notify.init("mpdevil")
+		self._notify=app.notify
```

Comment:
This `app.` syntax is nice. Where is this functionality documented?

Reply:
It's from `MainWindow.__init__()`. On `mpdevil.do_activate()` you're passing `mpdevil` class itself to `MainWindow`.

Reply:
Oh yes, I forgot about that its been there for a long time but I never used it.